src/core/nsfs.js

```diff
@@ -75,13 +76,14 @@ async function main(argv = minimist(process.argv.slice(2))) {
         }
         const http_port = Number(argv.http_port) || 6001;
         const https_port = Number(argv.https_port) || 6443;
+        const backend = argv.backend || (process.env.GPFS_DL_PATH ? 'GPFS' : '');
         const fs_root = argv._[0];
         if (!fs_root) return print_usage();
 
         let fs_config = {
             uid: Number(argv.uid) || process.getuid(),
             gid: Number(argv.gid) || process.getgid(),
-            backend: '',
+            backend: argv.backend || (process.env.GPFS_DL_PATH ? 'GPFS' : ''),
```

Comment:
no need to repeat, just use the const backend you defined before, with this shorthand syntax (this means `backend: backend,`) -
```suggestion
            backend,
```

Reply:
done